tests/ballerina-test/src/test/resources/test-src/structs/anon-struct.bal

```diff
@@ -1,25 +1,40 @@
-
 function testAnonStructAsFuncParam() returns (int) {
-    return testAnonStructFunc(10, {k:14, s:"sameera"});
+    return testAnonStructFunc(10, new (14, "sameera"));
 }
 
-function testAnonStructFunc(int i, type {int k = 10; string s;} anonSt) returns (int) {
+function testAnonStructFunc(
+    int i,
+    object {
+      public {
+          int k;
+          string s;
+      } new (k, s) {}
+     } anonSt ) returns (int) {
```

Comment:
Can you fix the indentations and the line breaks?
Previous formatting looks more readable..

Reply:
Please fix in other places as well

Reply:
Fixed